doc/ping.xml

```diff
@@ -851,6 +851,29 @@ xml:id="man.ping">
     hosts).</para>
   </refsection>
 
+  <refsection xml:id="id_collisions">
+    <info>
+      <title>ID COLLISIONS</title>
+    </info>
+    <para>
+    Unlike TCP and UDP, which use port to uniquely identify the recipient to
+    deliver data, ICMP uses identifier field (ID) for identification.
+    Therefore if on the same machine, at the same time, two ping processes
+    use the same ID, echo reply can be delivered to a wrong recipient.
+    This is known problem due to limited size of the 16 bit ID field.
```

Comment:
again minor nits:

This is a known problem due to the limited size of the 16-bit ID field